index.js

```diff
@@ -0,0 +1,50 @@
+var playItem = require('./src/playitem.js');
+
+var playlistMaker = function(player, plist) {
+  var currentIndex = 0;
+  var playlist = function playlist(list) {
+    this.playlist = playlistMaker(this, list);
+  };
+
+  playlist.list = function list() {
+    return plist.slice();
+  };
+
+  playlist.item = function item(index) {
```

Comment:
I wish there was a way to access playlist items without typing `.item()` all the time. I know you have moral objections to making the playlist plugin array-like, but what about overloading the plugin function?

```js
player.playlist(7); // returns item 7 in the playlist
```

Every time I have to work with a DOM API that requires the extra level of indirection (e.g. `video.buffered.start(0)`), I want to punch someone.